Tests/Unit/RealRemoteConfigTests.swift

```diff
@@ -11,6 +11,27 @@ class RealRemoteConfigSpec: QuickSpec {
                 expect(RealRemoteConfig.shared.getString("foo", "bar")).to(equal("moo"))
             }
         }
+        describe("getBoolean function") {
+            it("returns a boolean") {
```

Comment:
Maybe something like this is more clear?
`retrieves boolean from the cache for provided key`